test/recorder.spec.ts

```diff
@@ -60,6 +60,16 @@ describe('Recorder', function() {
     expect(recorderMetricsSpy).toHaveLength(1);
     expect(recorderMetricsSpy[0]).toEqual(expectedMetric);
   });
+  
+  it('recordActionInvoke()', async function() {
+    const recorderMetricsSpy = [];
+    const recorder = new Recorder('test', new testSink(), recorderMetricsSpy);
+    const expectedMetric = new Metric('test.action.invoked', MetricType.Counter, 1, { action: 'convert', success: true });
```

Comment:
I think adding `success` here might sound confusing, since you don't really have the data because the action didn't finished yet.

Reply:
Yeah, makes sense 👍